gym/lib/gym/generators/build_command_generator.rb

```diff
@@ -50,7 +50,8 @@ def actions
 
         actions = []
         actions << :clean if config[:clean]
-        actions << :archive
```

Comment:
@KrauseFx  😨 shame on me! sorry - i don't know how it even worked when testing the sample projects :(